packages/ui/src/components/Tabs/api.ts

```diff
@@ -0,0 +1,14 @@
+import { forceInject } from '@/util'
+import { InjectionKey, Ref } from 'vue'
+
+export interface TabsState {
+  active: Ref<number>
+  tabs: Ref<any[]>
+  selectTab: (tab: number) => void
+}
+
+export const TABS_STATE: InjectionKey<TabsState> = Symbol('tabsState')

```

Comment:
- Naming does not match with other similar implementations. If to follow the way how it is implemented in other components, here it should be `TabsPanelApi` interface & `TABS_PANEL_API_KEY` symbol.
- For such cases, when some component provides api to communicate with it, its better to go in a usual "vue" way - provide it as a readonly reactive store, not just raw refs.
- It is not okay that `tabs` field is typed as any, but here is another more general problem.